apps/worker/src/app/workflow/usecases/execute-bridge-job/execute-bridge-job.usecase.ts

```diff
@@ -137,8 +137,13 @@ export class ExecuteBridgeJob {
   private async findControlVariables(command: ExecuteBridgeJobCommand, workflow: NotificationTemplateEntity) {
     const controls = await this.controlVariablesRepository.findOne({
       _organizationId: command.organizationId,
-      _workflowId: workflow._id,
-      _stepId: command.job.step._id,
+      _environmentId: command.environmentId,
```

Comment:
This was missing, it could cause controls lookup to occur against the incorrect environment. We will fix this separately.

Reply:
Resolved in a separate PR.